e2e/dractions/disapp.go

```diff
@@ -62,6 +76,25 @@ func EnableProtectionDiscoveredApps(ctx types.TestContext) error {
 	return nil
 }
 
+func getRecipeRef(recipeType, name, appNS, mgmtNS string) *ramen.RecipeRef {
+	var recipeRef *ramen.RecipeRef
+
+	switch recipeType {
+	case "generate", "workload":
+		recipeRef = &ramen.RecipeRef{
+			Name:      name + "-recipe",
+			Namespace: appNS,
+		}
+	case "vm":
+		recipeRef = &ramen.RecipeRef{
+			Name:      "vm-recipe",
```

Comment:
This is using well known value the ramen depends on, right? Do we have a constant we can use in ramen api? Do we have a private constant in some ramen package? If we do we want to have a constant in this file (likey the recipes package) and add a link the ramen constant.